packages/nx/src/project-graph/file-utils.ts

```diff
@@ -119,20 +119,28 @@ function defaultReadFileAtRevision(
   }
 }
 
+/**
+ * TODO(v18): Remove this function
+ * @deprecated
```

Comment:
Please add what to use instead and when this will be removed.

Reply:
added